org.adempiere.base/src/org/compiere/model/MInvoice.java

```diff
@@ -2545,12 +2545,22 @@ public boolean voidIt()
 			else
 				return reverseCorrectIt();
 		}
+		
+		// update taxes
+		MInvoiceTax[] taxes = getTaxes(true);
+		for (MInvoiceTax tax : taxes )
+		{
+			if ( !(tax.calculateTaxFromLines() && tax.save()) )
+				return false;
+		}

```

Comment:
Should just clear all invoice tax records instead of recalculate here (there's nothing to re-calculate here).